designs/odcr.md

```diff
@@ -0,0 +1,121 @@
+# On demand capacity reservation
```

Comment:
Have we thought at all about how we model pricing for instance types with ODCRs and the interactions that this might have with consolidation?

Reply:
I have not, thats a good question. Let me give it some thought.

Reply:
Updated with pricing for ODCR